README.md

```diff
@@ -452,15 +452,27 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 4.3 Detect code issues with a linter
```

Comment:
Is it really important to move down all list (within all translations)?

Reply:
@oshliaer Good question, maybe @BrunoScheufler and @js-kyle can chime-in

I find the order of advice to be important, the first bullets deal with the test structure and the basics. The last bullets deal with high-level stuff like CI. I don't find a reason to mix the topics... on the other end changing the numbers might break links

What are your thoughts?

Reply:
I really agree that new bullets should be added at the bottom of the section, it's very confusing to shuffle things around when introducing a new best practice

However, yes, I understand there is great value in shuffling things around in the sections, I just thing that should be done in a separate PR so it is clear, and ensure that all the translations are also shuffled at the same time where possible 👍

Reply:
@js-kyle Resonates with me, will separate into a different PR next time